source/Utils/src/FilterTracks.cc

```diff
@@ -149,7 +174,8 @@ void FilterTracks::processEvent( LCEvent * evt )
 	 nhitinner    > _NHitsInner  &&
 	 nhitouter    > _NHitsOuter  &&
 	 pt           > _MinPt       &&
-	 chi2spatial  > _Chi2Spatial)
+	 chi2spatial  > _Chi2Spatial &&
+   nholes       <= _MaxHoles)
```

Comment:
minor nitpick here that the formatting is slightly off.

Reply:
Fixed